Weather/Weather/Extensions/UIViewExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIViewExtensions.swift
+//  Weather
+//
+//  Created by minh duc on 2/19/19.
+//  Copyright © 2019 minhduc. All rights reserved.
+//
+
+import UIKit
+
+extension UIView {
+    convenience init(height: CGFloat) {
+        self.init(frame: CGRect(x: 0, y: 0, width: 0, height: height))
+    }
+}
+
+extension CGRect {
```

Comment:
extension của mỗi phần tách riêng 1 file khác nhau, và khi init với width = 0 thì nó sẽ không hiện lên đâu

Reply:
init width = 0 với tableViewHeader sẽ tự tính toán width nên không cần width.